### PR TITLE
fix issues 113 & 114

### PR DESCRIPTION
**Pull request by gucong3000**
https://github.com/mikaelbr/node-notifier/issues/114 fixed
https://github.com/mikaelbr/node-notifier/issues/113 fixed

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -161,20 +161,20 @@ module.exports.mapToMac = function (options) {
...
-    if(resultantData) {
-      resultantData = resultantData.toLowerCase().trim();
-      if(resultantData.match(/^activate/)) {
-        resultantData = 'activate';
-      }
-    }
-    fn.apply(emitter, [err, resultantData]);
-    if (err || !mapper || !resultantData) return;
-
-    var key = mapper(resultantData);
```
Comment (gucong3000) [inline lib/utils.js]: https://github.com/mikaelbr/node-notifier/pull/115/files#diff-50e3aa130a4f97a42ee2cf111c7b1d9dL164
mix EOL here
`git config --global core.safecrlf true`

see: http://adaptivepatchwork.com/2012/03/01/mind-the-end-of-your-line/

**Comment by gucong3000** [comment]
@mikaelbr Will you merge it?

**Comment by mikaelbr** [comment]
Sorry about taking so long. Thanks for contributing!

**Comment by gucong3000** [comment]
:+1:

**Pull request merged**